### ISSUE TITLE
Bulk CSV Import for cables with multiple terminations

### ISSUE THREAD

**Issue by clatini** [netbox-community/netbox#10319]
### NetBox version

v.3.3.2

### Feature type

Change to existing functionality

### Proposed functionality

At this time there is no way to bulk import cables when one or both sides have multiple terminations

### Use case

for example, adding a special separator in 'device_x_name' column, for example a '|':

```
side_a_device,side_a_type,side_a_name,side_b_device,side_b_type,side_b_name
Switch1,dcim.interface,Eth1,PatchPanel1,dcim.frontport,port1|port2|port3|port4
```

### Database changes

n/a

### External dependencies

n/a

**Comment by jeremystretch** [comment]
The implementation for this should match whatever approach we use for #7376 as far as the CSV formatting for multiple objects.

**Comment by jeremystretch** [comment]
This is probably more viable now that we've implemented JSON/YAML imports for cables under #4347 in NetBox v3.4.
